GeorgianMemenator/Scenes/CatTagsList/CatTagsListViewModel.swift

```diff
@@ -12,3 +12,10 @@ import Foundation
 class CatTagsListViewModel: ObservableObject {
     
 }
+
+var tags: [CatTagDetail] = [
```

Comment:
The tags should be part of the `CatTagsListViewModel`. Now it is global variable

Reply:
Also it needs to have @Published annotation, please read:

https://www.hackingwithswift.com/quick-start/swiftui/what-is-the-published-property-wrapper